overlord/state/taskrunner.go

```diff
@@ -231,7 +231,7 @@ func (r *TaskRunner) Ensure() {
 			// Dependencies still unhandled.
 			continue
 		}
-		logger.Debugf("Running task %s: %s", t.ID(), t.Summary())
+		logger.Debugf("Running task %s: %q %s", t.ID(), t.Status(), t.Summary())
```

Comment:
%q is unnecessary here since the strings are very simple. Perhaps something like this:

`"Running task %s on %s: %s"`

LGTM